Week3/homework/1-step3.js

```diff
@@ -1,11 +1,13 @@
 'use strict';
 
 function foo(func) {
-  // What to do here? 
+  // What to do here?
+  func();
 }
 
```

Comment:
It is generally a good idea to put the func call in a return statement.  You don't know if it is going to return a value that should be returned from your function.  In things like filter, map and reduce it is.
`return func();`
Would be a better idea.